### ISSUE TITLE
a11y: can't log in or access user profile from the keyboard (WCAG level A)

### ISSUE THREAD

**Issue by eatyourgreens** [OxfordRSE/gutenberg#403]
If I tab through the top menu, I can open the user account popup menu, but tab then closes the menu and navigates to 'Licensing & Attribution'. I can't navigate into the popup menu from the keyboard.

https://github.com/user-attachments/assets/b677ef9a-45c6-4c4b-ba13-d63393419827

Relevant WCAG SC: [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG22/Understanding/keyboard.html) (level A.)

**Comment by eatyourgreens** [comment]
Turns out the first dropdown item is focused when the menu opens, but it's not visually obvious that focus has jumped to the menu. That might fail at level AA ([2.4.7 Focus Visible](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html).)

So you can log in by tabbing to the user avatar button, pressing Enter to open the menu, then Enter again to activate the Sign-in menu item.

Once you are logged in, the user profile link is still inaccessible, because dropdown headers aren't accessible from the keyboard.
